unicore/distribute/api/repos.py

```diff
@@ -73,7 +104,7 @@ def post(self):
                     'url': self.request.route_url('repositoryresource',
                                                   name=name)
                 }))
-        return format_diffindex(changes)
+        return ''
```

Comment:
Only because the workspace doesn't return the changes. I'm happy to update elastic-git